qemu/tests/cfg/block_multifunction.cfg

```diff
@@ -0,0 +1,11 @@
+- block_multifunction:
+    type = block_multifunction
+    only x86_64,i386,ppc64,ppc64le
+    only virtio_blk, virtio_scsi, spapr_vscsi
```

Comment:
Confirm with the function owner xuma that spapr-vscis does not support multifunction function.

Reply:
Thanks @zhenyzha removed spapr_vscsi